locales/en/index.yml

```
@@ -495,8 +495,14 @@ authentication:
       incorrectCiePinContent1: For your security, you can try entering the card PIN no more than three times. Then you will have to unlock the card by entering the PUK.
       incorrectCiePinContent2: Keep in mind that the PIN code of your Electronic Identity Card (CIE) is composed of 8 digits. Four digits that were given to you at the registry office and another four digits that were sent to you at home, along with the card.
       alert: Are you sure you want to go back?
+      subtitleHelp: The PIN code consists of 8 digits.
+      subtitleCTA: Where can I find it?
+      bottomSheetTitle: Where can I find my PIN?
+      bottomSheetText: "The PIN code consists of 8 digits:\n• the first 4 digits you received at the registry office;\n• the last 4 digits you received at home, along with your electronic ID card.\n**Please enter them following this order.**"
```

The same as for Italian:
```suggestion
      bottomSheetText: "The PIN code consists of 8 digits:\n\n• the first 4 digits you received at the registry office;\n\n• the last 4 digits you received at home, along with your electronic ID card.\n\n**Please enter them following this order.**"
```